Translation to Spanish [#1] 300 Words

Santu41 opened the pull request:
Esta es mi primera colaboración en el proyecto, espero sea de su agrado, mi columna esta completamente traducida al español y espero que me asignen una nueva para continuar trabajando! :+1:

Santu41 committed:
Translation to Spanish [300 Worlds] mixins.md

Aquí dejo mi colaboración en el proyecto, la columna que me ha sido asignada esta traducida completamente al español.

Santu41 committed:
Update mixins.md

miljan-aleksic commented:
Muchas gracias por tu colaboración @Santu41, en breve lo reviso. Mientras te he asigndo Plugins (Advanced).

Santu41 committed:
Translation to Spanish [264 Words] plugins.md

Aquí dejo mi segunda colaboración en el proyecto, la columna que me ha sido asignada esta traducida completamente al español.

Santu41 commented:
@miljan-aleksic cometí un error en el titulo de la descripción de la traducción (Worlds - Words) ese es el cambio que debería hacerse pero no tengo los permisos necesarios, saludos!
